bitnami/chainloop/templates/controlplane/deployment.yaml

```
@@ -0,0 +1,220 @@
+        checksum/secret-config: {{ include (print $.Template.BasePath "/controlplane" "/secret-config.yaml") . | sha256sum }}
+        checksum/cas-private-key: {{ include (print $.Template.BasePath "/controlplane" "/secret-jwt-cas-private-key.yaml") . | sha256sum }}
+        kubectl.kubernetes.io/default-container: controlplane
+      labels: {{- include "chainloop.controlplane.labels" . | nindent 8 }}
+    spec:
+      {{- include "common.images.renderPullSecrets" (dict "images" (list .Values.controlplane.image .Values.controlplane.migration.image) "context" $) | nindent 6 }}
+      serviceAccountName: {{ include "controlplane.serviceAccountName" . }}
+      automountServiceAccountToken: {{ .Values.controlplane.automountServiceAccountToken }}
+      {{- if .Values.controlplane.hostAliases }}
+      hostAliases: {{- include "common.tplvalues.render" (dict "value" .Values.controlplane.hostAliases "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.controlplane.affinity }}
+      affinity: {{- include "common.tplvalues.render" ( dict "value" .Values.controlplane.affinity "context" $) | nindent 8 }}
+      {{- else }}
+      affinity:
+        podAffinity: {{- include "common.affinities.pods" (dict "type" .Values.controlplane.podAffinityPreset "component" "controlplane" "customLabels" $podLabels "context" $) | nindent 10 }}
+        podAntiAffinity: {{- include "common.affinities.pods" (dict "type" .Values.controlplane.podAntiAffinityPreset "component" "controlplane" "customLabels" $podLabels "context" $) | nindent 10 }}
+        nodeAffinity: {{- include "common.affinities.nodes" (dict "type" .Values.controlplane.nodeAffinityPreset.type "key" .Values.controlplane.nodeAffinityPreset.key "values" .Values.controlplane.nodeAffinityPreset.values) | nindent 10 }}
+      {{- end }}
+      {{- if .Values.controlplane.nodeSelector }}
+      nodeSelector: {{- include "common.tplvalues.render" ( dict "value" .Values.controlplane.nodeSelector "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.controlplane.tolerations }}
+      tolerations: {{- include "common.tplvalues.render" (dict "value" .Values.controlplane.tolerations "context" .) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.controlplane.priorityClassName }}
+      priorityClassName: {{ .Values.controlplane.priorityClassName | quote }}
+      {{- end }}
+      {{- if .Values.controlplane.schedulerName }}
+      schedulerName: {{ .Values.controlplane.schedulerName | quote }}
+      {{- end }}
+      {{- if .Values.controlplane.topologySpreadConstraints }}
+      topologySpreadConstraints: {{- include "common.tplvalues.render" (dict "value" .Values.controlplane.topologySpreadConstraints "context" .) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.controlplane.podSecurityContext.enabled }}
+      securityContext: {{- omit .Values.controlplane.podSecurityContext "enabled" | toYaml | nindent 8 }}
```

I'm sorry I know this comes from a typo in the template:
```suggestion
          securityContext: {{- include "common.compatibility.renderSecurityContext" (dict "secContext" .Values.controlplane.podSecurityContext "context" $) | nindent 8 }}
```